File: 1-Iniciante/ex1114.c
#include <stdio.h>
#define VALIDA 2002

int login(int senha);

int main(){
    /*variaveis*/
    int i;
    int senha;

    /*processamento*/
    while(1){
        scanf("%d",&senha);
        login(senha);

        if(login(senha) == 1){
            break;
        }
        printf("Senha Invalida\n");
    }

    /*saida*/
    printf("Acesso Permitido\n");
    return 0;
}

int login(int senha){
    if(senha == VALIDA){
        return 1;
    }
    else{
        return 0;
    }
}

File: 1-Iniciante/ex1114.js
// entrada
let input = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd

// processamento & saida
let segredo = '2002';
let senha_dig = input[0];
let pos = 1;
while(senha_dig != segredo){
    console.log('Senha Invalida');
    senha_dig = input[pos++];
}
console.log('Acesso Permitido');